examples/custom-json-modules/webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const toml = require("toml");
+const json5 = require("json5");
+const yaml = require("yamljs");
+
+module.exports = {
+	module: {
+		rules: [
+			{
+				test: /\.toml$/,
+				type: "json",
+				parser: {
+					parse(input) {
```

Comment:
Can we optimize this to

``` js
parser: {
  parse: toml.parse
}
```

This would make it a little bit easier to use.

Maybe we need to drop the second argument of `parse` for this.

Reply:
Sometimes you need to specify options for the parser, so we still need function support

Reply:
Agree with @evilebottnawi

Reply:
Or maybe we should support:
parser: toml.parse
and
parse.parse(input, context) {}

Reply:
Or maybe we should support:
parser: toml.parse
and
parse.parse(input, context) {}

Reply:
sometimes you may want to change something in the code before parse, so i think will be great supports these:
`parse: toml.parse`
`parse: (input) => { /* Many logic around `input` */ return toml.parse(input) }`

Reply:
> parse: toml.parse

In this case we must remove the second parameter from parse function
I think that it is not flexible

Reply:
I think that this one will be great:
parser.parse: toml.parse // parse has only one parameter - an input
and
parser(input, context) {}

Reply:
I think that this one will be great:
parser.parse: toml.parse // parse has only one parameter - an input
and
parser(input, context) {}